api/pom.xml

```diff
@@ -341,6 +341,8 @@
                         <Bundle-Description>Jakarta JSON Processing API ${spec.version}</Bundle-Description>
                         <Specification-Vendor>${vendor.name}</Specification-Vendor>
                         <Implementation-Build-Id>${buildNumber}</Implementation-Build-Id>
+                        <Import-Package></Import-Package>
+                        <Require-Capability>osgi.extender;filter:="(osgi.extender=osgi.serviceloader.processor)", osgi.serviceloader; filter:="(osgi.serviceloader=jakarta.json.spi.JsonProvider)"; cardinality:=multiple</Require-Capability>
```

Comment:
Is that osgi extender required? can we avoid using it?

Reply:
I mean follow https://docs.osgi.org/reference/portable-java-contracts.html somehow